zerver/lib/create_user.py

```diff
@@ -14,6 +14,41 @@ def random_api_key() -> str:
     altchars = ''.join([choices[ord(os.urandom(1)) % 62] for _ in range(2)]).encode("utf-8")
     return base64.b64encode(os.urandom(24), altchars=altchars).decode("utf-8")
 
+def copy_user_settings(source_profile, target_profile) -> UserProfile:
+    target_profile.enable_stream_desktop_notifications = source_profile.enable_stream_desktop_notifications
+    target_profile.enable_stream_email_notifications = source_profile.enable_stream_email_notifications
+    target_profile.enable_stream_push_notifications = source_profile.enable_stream_push_notifications
+    target_profile.enable_stream_sounds = source_profile.enable_stream_sounds
+
+    target_profile.enable_desktop_notifications = source_profile.enable_desktop_notifications
+    target_profile.pm_content_in_desktop_notifications = source_profile.pm_content_in_desktop_notifications
+    target_profile.enable_sounds = source_profile.enable_sounds
+    target_profile.enable_offline_email_notifications = source_profile.enable_offline_email_notifications
+    target_profile.message_content_in_email_notifications = \
+        source_profile.message_content_in_email_notifications
+    target_profile.enable_offline_push_notifications = source_profile.enable_offline_push_notifications
+    target_profile.enable_online_push_notifications = source_profile.enable_online_push_notifications
+    target_profile.enable_digest_emails = source_profile.enable_digest_emails
+    target_profile.realm_name_in_notifications = source_profile.realm_name_in_notifications
```

Comment:
This should probably loop over the `property_types` for `display_settings` and `notification_settings` to make it less annoying to maintain and easier to verify.